integration-tests/src/test/java/eu/tsystems/mms/tic/testframework/test/webdrivermanager/WebDriverManagerTest.java

```diff
@@ -198,7 +227,7 @@ public void testT07a_WindowSizeRequest() {
     @Test
     public void testT08_invalidWindowSize() {
         PROPERTY_MANAGER.setTestLocalProperty(DesktopWebDriverRequest.Properties.WINDOW_SIZE, "katze");
-        String property = PROPERTY_MANAGER.getProperty(DesktopWebDriverRequest.Properties.WINDOW_SIZE, PROPERTY_MANAGER.getProperty(DesktopWebDriverRequest.Properties.DISPLAY_RESOLUTION));
+        String property = PROPERTY_MANAGER.getProperty(DesktopWebDriverRequest.Properties.WINDOW_SIZE, PROPERTY_MANAGER.getProperty(DesktopWebDriverRequest.Properties.WINDOW_SIZE));
```

Comment:
I think this is wrong: If we _need_ the default value, we _cannot have_ the default value. (The default value has to be different from the non-default value to work.)

... If this value must be present, I think the line should only read
`final String property = PROPERTY_MANAGER.getProperty(DesktopWebDriverRequest.Properties.WINDOW_SIZE);`, if it can be absent, then I think it's broken but not sure how it should be corrected if the old version was not acceptable.

Reply:
Looks like a bug to me.
`WINDOW_SIZE` is the replacement for `@deprecated` `DISPLAY_RESOLUTION`.
Most projects only have `DISPLAY_RESOLUTION` configured, thats why is working as a fallback.

Reply:
Yes, for that test there is no need to define a default for `getProperty(...
)`.